patches/react-remove-scroll+2.6.3.patch

```diff
@@ -34,35 +41,17 @@ index 08eda83..e48ccd6 100644
 -        return;
 -    }, [props.inert, props.lockRef.current, props.shards]);
      var shouldCancelEvent = React.useCallback(function (event, parent) {
-         if ('touches' in event && event.touches.length === 2) {
+         if (('touches' in event && event.touches.length === 2) || (event.type === 'wheel' && event.ctrlKey)) {
              return !lastProps.current.allowPinchZoom;
-@@ -139,8 +123,5 @@ export function RemoveScrollSideCar(props) {
+@@ -139,10 +121,7 @@ export function RemoveScrollSideCar(props) {
              document.removeEventListener('touchstart', scrollTouchStart, nonPassive);
          };
      }, []);
 -    var removeScrollBar = props.removeScrollBar, inert = props.inert;
 -    return (React.createElement(React.Fragment, null,
 -        inert ? React.createElement(Style, { styles: generateStyle(id) }) : null,
--        removeScrollBar ? React.createElement(RemoveScrollBar, { gapMode: "margin" }) : null));
+-        removeScrollBar ? React.createElement(RemoveScrollBar, { gapMode: props.gapMode }) : null));
 +    return (React.createElement(React.Fragment, null));
  }
-diff --git a/node_modules/react-remove-scroll/dist/es2015/UI.js b/node_modules/react-remove-scroll/dist/es2015/UI.js
-index 26c94a8..75d91ae 100644
---- a/node_modules/react-remove-scroll/dist/es2015/UI.js
-+++ b/node_modules/react-remove-scroll/dist/es2015/UI.js
-@@ -1,6 +1,5 @@
- import { __assign, __rest } from "tslib";
- import * as React from 'react';
--import { fullWidthClassName, zeroRightClassName } from 'react-remove-scroll-bar/constants';
- import { useMergeRefs } from 'use-callback-ref';
- import { effectCar } from './medium';
- var nothing = function () {
-@@ -29,8 +28,4 @@ RemoveScroll.defaultProps = {
-     removeScrollBar: true,
-     inert: false,
- };
--RemoveScroll.classNames = {
--    fullWidth: fullWidthClassName,
--    zeroRight: zeroRightClassName,
--};
```

Comment:
This existing bit of the patch wasn't necessary. These lines do not set classes, they set a value `RemoveScroll.classNames` that is imported by other things and used for stuff.